snippets/apply_vaccine.py

```diff
@@ -0,0 +1,33 @@
+from vaccine_centers_db import get_center_data
+import collections
+from population import generate_random_population
+
+
+class OutOfDosesException(Exception):
+    pass
+
+
+def lookup_vaccine_name(individual):
+    location_day_data = get_center_data(individual.jab_location, individual.date_first_jab)
+    return location_day_data.vaccine_name
+
+def apply_vaccine_to_population(population, ndoses, threshold_age):
+    for ind in population:
+        if ndoses > 0:
+            if (ind["age"] > threshold_age or ind["isAtRisk"]):
+                if ind["hasFirstJab"]:
+                    inject_second_jab(ind)
+                else:
+                    inject_first_jab(ind)
+            ndoses = ndoses - 1
```

Comment:
this line should be indented as now ndoses gets reduced by 1 regardless of what happens in the if